rmw_fastrtps_cpp/src/rmw_node.cpp

```diff
@@ -37,18 +38,22 @@ rmw_create_node(
   const char * name,
   const char * namespace_,
   size_t domain_id,
-  const rmw_node_security_options_t * security_options,
+  const rmw_security_options_t * security_options,
   bool localhost_only)
 {
+  (void)domain_id;
+  (void)security_options;
+  (void)localhost_only;
```

Comment:
I moved localhost logic to the participant, and I haven't modified node API yet.

Reply:
Localhost logic is now a context configuration, not a node one.
I will not change the API as part of the first iteration, to avoid modifying others rmw implementations.